spring-cloud-app-broker-integration-tests/src/test/java/org/springframework/cloud/appbroker/integration/WiremockComponentTest.java

```diff
@@ -47,7 +47,8 @@
 		"spring.cloud.appbroker.deployer.cloudfoundry.default-org=test",
 		"spring.cloud.appbroker.deployer.cloudfoundry.default-space=development",
 		"spring.cloud.appbroker.deployer.cloudfoundry.secure=false",
-		"spring.cloud.appbroker.deployer.cloudfoundry.properties.buildpack=example-buildpack"
+		"spring.cloud.appbroker.deployer.cloudfoundry.properties.buildpack=example-buildpack",
+		"spring.cloud.appbroker.deployer.cloudfoundry.properties.buildpack=cflinuxfs4"
```

Comment:
Was this meant to be stack, not buildpack?

Reply:
Oh, I should have that deleted. I may add the (right) property there if I can't create a good AT.
Thanks!